src/components/common/AddEditOrgPerson.vue

```diff
@@ -114,55 +114,125 @@
 
             <!-- Roles -->
             <article class="mt-2">
-              <div class="font-weight-bold">Roles</div>
-              <v-card flat rounded="sm" class="gray-card mt-4 px-4">
-                <v-row class="align-center">
-                  <v-col cols="4" v-if="showCompletingPartyRole" class="py-0">
-                    <v-checkbox
-                      id="cp-checkbox"
-                      class="mt-5"
-                      v-model="selectedRoles"
-                      :value="RoleTypes.COMPLETING_PARTY"
-                      :label="RoleTypes.COMPLETING_PARTY"
-                      :disabled="disableCompletingPartyRole"
-                      @change="assignCompletingPartyRole()"
-                    />
-                  </v-col>
+              <div class="font-weight-bold">{{ getPeopleAndRolesResource.rolesTitle || 'Roles' }}</div>
+              <div v-if="getPeopleAndRolesResource.rolesSubtitle">
+                {{ getPeopleAndRolesResource.rolesSubtitle }}
+              </div>
+              <div class="form__row three-column mt-4">
+                <v-card flat rounded="sm" class="item gray-card px-4">
+                  <v-row no-gutters class="align-center mt-4">
+
+                    <v-col cols="4" v-if="showCompletingPartyRole">
+                      <v-checkbox
+                        id="cp-checkbox"
+                        class="mt-0"
+                        v-model="selectedRoles"
+                        :value="RoleTypes.COMPLETING_PARTY"
+                        :label="RoleTypes.COMPLETING_PARTY"
+                        :disabled="disableCompletingPartyRole"
+                        :rules="enableRules ? roleRules: []"
```

Comment:
This checkbox is only enabled for staff (ie, users can't uncheck it), but I added the rules to prevent staff errors, ie:

![image](https://user-images.githubusercontent.com/10002889/212782956-71acb37e-e47f-4cca-a978-9f0d153e04b8.png)